.github/workflows/exclude-file.txt

```diff
@@ -0,0 +1,5 @@
+twitter.com
```

Comment:
Where did these excludes come from?

Reply:
by running the link checker and noting what links are failing :D these are the links you don't wanna check or avoid 
for eg., i've added `github.com` and `raw.githubusercontent.com` to avoid rate limited.

Reply:
Works for me :)